gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [libraries]
-android-gradle-plugin = { module = "com.android.tools.build:gradle", version = "7.4.0-beta02" }
+android-gradle-plugin = { module = "com.android.tools.build:gradle", version = "7.4.1" }
```

Comment:
was the project compilable on `IntelliJ IDEA 2022.3.2`?

Reply:
tested with IntelliJ IDEA 2023.2 (Community Edition)
Is there a reason for it to be compilable on the older IDEA?